core/templates/pages/about-page/charts/bar-chart.component.css

```
@@ -4,14 +4,14 @@
   https://rtlcss.com/learn/usage-guide/control-directives .
 */
 
-
 .oppia-bar-chart .charts-css.bar {
   --labels-size: 220px;
+  margin-left: var(--labels-size); 
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)